examples/step-64/step-64.cu

```diff
@@ -236,17 +270,22 @@ namespace Step64
     const QGauss<1> quad(fe_degree + 1);
     mf_data.reinit(mapping, dof_handler, constraints, quad, additional_data);
 
-    // We need to store the value of the coefficient for each quadrature point
-    // in every locally owned cells.
+
     const unsigned int n_owned_cells =
-      std::distance<decltype(dof_handler.begin_active())>(
-        dof_handler.begin_active(), dof_handler.end());
+      dynamic_cast<const parallel::Triangulation<dim> *>(
+        &dof_handler.get_triangulation())
+        ->n_locally_owned_active_cells();
```

Comment:
I am confused: In general, `n_locally_owned_active_cells()` is less than `std::distance(...)`. Was the code wrong before?

Reply:
Yes, the size was larger than necessary.